aw_watcher_window/printAppStatus.jxa

```diff
@@ -23,37 +23,73 @@ var url = undefined, incognito = undefined, title = undefined;
 switch(appName) {
   case "Safari":
     // incognito is not available via safari applescript
+    try { // see catch for details
     url = Application(appName).documents[0].url();
-    title = Application(appName).documents[0].name();
+	  title = Application(appName).documents[0].name();
+    } catch {
+      // On Mac, you can have an application open without a window.
+      // You can also have safari open with just a Devtools window, meaning no documents
+      // This edge-case previously caused an error meaning time wasn't tracked.
+      // I'm not 100% that a title of "No window" is the right approach, but it is the clearest description of what's happening I can see right now.
+      title = "No window" 
```

Comment:
I think a single comment is fine, but perhaps lift out the "No window" constant into a variable, and document the behavior by the declaration/first use?